src/app/public/modules/status-indicator/status-indicator.component.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input
+} from '@angular/core';
+
+import {
+  SkyIndicatorIconUtility
+} from '../shared/indicator-icon-utility';
+
+import {
+  SkyIconStackItem
+} from '../icon/icon-stack-item';
+
+const INDICATOR_TYPE_DEFAULT = 'warning';
+
+@Component({
+  selector: 'sky-status-indicator',
+  templateUrl: './status-indicator.component.html',
+  styleUrls: ['./status-indicator.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SkyStatusIndicatorComponent {
+  /**
+   * Specifies a style for the status indicator to determine the icon.
+   * The valid options are `danger`, `info`, `success`, and `warning`.
+   * @default "warning"
+   */
+  @Input()
+  public set indicatorType(value: string) {
```

Comment:
Should the input value be a literal string type so intellisense would enforce proper values? It would also get picked up by jsDocs for our automated documentation. 
```
type SkyIndicatorType = 'danger' | 'info' ...
```
We do this for a [couple other components](https://developer.blackbaud.com/skyux-popovers/docs/dropdown#type-alias-skydropdowntriggertype), so I'm not really sure which approach is best. Any thoughts?

Reply:
Good suggestion. I had started down that route but couldn't go all in because alert (which shares logic with status indicator through the `SkyIndicatorIconUtility`) uses `string` for its `alertType` property, and I can't change that type without breaking compatibility. I went ahead and implemented it anyway and will just cast `alertType` when using `SkyIndicatorIconUtility`.